modules/tinymce/CHANGELOG.md

```diff
@@ -6,10 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+- New `pad_empty_with_br` option that can be set to `true` to pad empty block elements with `<br>` tag instead of nbsp character. #TINY-9861
```

Comment:
Since this is going into a patch release, maybe we reword it as restoring something that shouldn't have been removed 😅

Reply:
Yeah, that makes sense. Brian will reword this when he does the changelog edits